src/Characters/equipment.py

```diff
@@ -1,12 +1,31 @@
+from dataclasses import dataclass
+from enum import Enum
+
+class EquipmentType(Enum):
+    Weapon = 1
+    Armour = 2
+
+class EquipmentRarity(Enum):
+    Common = 1
+    Rare = 2
+    Epic = 3
+    Legendary = 4
+
+@dataclass
 class Equipment():
-    def __init__(self, type :str, stats :list, description :str):
+    def __init__(self, type :EquipmentType = EquipmentType.Weapon,
```

Comment:
This constructor is probably not necessary anymore, because this is now dataclass.

Reply:
Yeah, you're right. Though I rather removed the dataclass, just remain consistent.